docs/source/conf.py

```diff
@@ -168,7 +168,11 @@ def setup(app):
 
 # -- Linkcode configuration --------------------------------------------------
 DEFAULT_REF = "dev"
-if os.environ.get("GITHUB_REF_TYPE", "branch") == "tag":
+if git_ref := os.environ.get("READTHEDOCS_GIT_IDENTIFIER", None):
```

Comment:
We don't use the walrus operator anywhere else in MONAI so I'd suggest not using it here to be consistent with our code standard. Thanks!

Reply:
Ok, no problem. Then I can get rid of the weird noop assignment in the next line as well